docker/Dockerfile

```diff
@@ -9,6 +9,7 @@ COPY build/* /
 RUN addgroup -g 1000 pgo
 RUN adduser -D -u 1000 -G pgo -g 'Postgres Operator' pgo
 
-USER 1000:1000
+USER pgo:pgo
```

Comment:
Hello, having "named" user prevents kubernetes to check for non root. I think it would be best to keep as it's now.